money-spring3/src/main/scala/com/comcast/money/spring3/MoneyClientHttpRequestInterceptor.scala

```diff
@@ -69,9 +69,11 @@ class MoneyClientHttpRequestInterceptor extends ClientHttpRequestInterceptor {
     SpanLocal.current foreach { span =>
       val headers = httpRequest.getHeaders
       headers.add("X-MoneyTrace", Formatters.toHttpHeader(span.info.id))
-      headers.add("X-B3-TraceId", span.info.id.traceId)
-      headers.add("X-B3-ParentSpanId", span.info.id.parentId.toString)
-      headers.add("X-B3-SpanId", span.info.id.selfId.toString)
+      Formatters.toB3Headers(span.info.id)(
```

Comment:
Since this function gives no control over what's set I would prefer the knowledge be kept in the Formatters I think this will prevent other locations in the library setting different header names.

`Formatters.toB3Headers(span.info.id, headers.add)`

Reply:
Agree, but we should also change fromB3Header and from/toHttpHeader. (Which should probably be renamed to from/toMoneyHeader)